src/extension/module.js

```diff
@@ -433,33 +433,39 @@
   /**
    * Recognizes a SharePoint URL.
    * @private
+   * @param {Sidekick} sk The sidekick
    * @param {URL} url The URL
    * @returns {boolean} {@code true} if URL is SharePoint, else {@code false}
    */
-  function isSharePoint(url) {
-    return /\w+\.sharepoint.com$/.test(url.host);
+  function isSharePoint(sk, url) {
+    const { host } = url;
+    const { config: { mountpoint } } = sk;
+    return /\w+\.sharepoint.com$/.test(host)
+      || (mountpoint && new URL(mountpoint).host === host && !host.endsWith('.google.com'));
```

Comment:
detail: `.endsWith()` is computationally cheaper than `new URL().host` so I would move it before:

```suggestion
      || (mountpoint && !host.endsWith('.google.com') && new URL(mountpoint).host === host);
```